edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_footer.scss

```diff
@@ -1,257 +1,174 @@
-// Open edX: LMS footer
-// ====================
-@import '../base/grid-settings';
-@import 'neat/neat'; // lib - Neat
+.wrapper-footer footer#footer-openedx {
+
+ a, .copyright,
+ .colophon .nav-colophon li a {
+    color: $inverse;
+  }
+}
+
+.vue-footer {
+  background-color: $vue-header-footer !important;
+}
+
+.apprenticeships-footer {
+   background-color: $app-header-footer !important;
+}
+
+ .wrapper-footer footer#footer-openedx .colophon {
+    float: none;
+    margin: auto;
+    min-height: 30px;
+  }
+
+.wrapper-footer footer#footer-openedx .footer-about-openedx:last-child {
+  margin: auto;
+  float: none;
+  width: 85%;
+  margin-top: 20px;
+}
+
+.wrapper-footer footer#footer-openedx .footer-about-openedx {
+  margin: auto;
+  width: 100%;
+}
+
+div.wrapper-footer footer#footer-openedx .footer-about-openedx a{
+  padding-left:30px;
+  float:left;
```

Comment:
Please add a space before the value assignment. @luismorenolopera
`float: left;`

Reply:
done